telegram/_bot.py

```diff
@@ -8867,7 +8867,7 @@ async def replace_sticker_in_set(
             api_kwargs=api_kwargs,
         )
 
-    def to_dict(self, recursive: bool = True) -> JSONDict:
```

Comment:
I don't think this makes sense. If you have an `obj` and only know that it's an instance of `TelegramObject`, you want to be able to call `obj.to_dict(recrusive=False)`. if `isinstance(obj, Bot)`, that won't work this way.